sdk/test-utils/test-utils-vitest/test/tracing/mockInstrumenterAssert.spec.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createTracingClient, TracingClient, useInstrumenter } from "@azure/core-tracing";
+import { MockTracingSpan, MockInstrumenter } from "../../src/index.js";
+import { chaiAzure } from "../../src/azureAssert.js";
+import { MockContext } from "../../src/tracing/mockContext.js";
+import { OperationTracingOptions } from "@azure/core-tracing";
+import { describe, it, beforeEach, chai } from "vitest";
+chai.use(chaiAzure);
```

Comment:
nit: should we use expect.extend to ensure it works in the vitest context? I know that we can also use `chai` from `vitest` but I think we'd prefer to use `expect.extend` to avoid confusion, so should the tests here try that?

Reply:
We have that supported in another test suite: https://github.com/Azure/azure-sdk-for-js/blob/73c622fbdefba4779a32e8d0620d6e84728f3460/sdk/test-utils/test-utils-vitest/test/tracing/mockInstrumenterExpect.spec.ts#L11-L12